OpenTrons/opentrons-api
# fix(app): fix view network details data fetch issue

# Overview
The component missed fetch the network interface after rendering the page and fix that issue
close RQA-1274

![Screenshot 2023-08-11 at 5 35 25 PM](https://github.com/Opentrons/opentrons/assets/474225/b1ea511f-613a-4bcc-a7de-fd97db6acea4)

tested on a Flex and on my dev kit(initialize my dev kit)



# Test Plan
1. disconnect wifi (RobotSettings Network settings -> select any ssid and put invalid password) We will add disconnect function to Touchscreen app soon
2. change `/data/ODD/config.json`:  `unfinishedUnboxingFlowRoute` null -> "/welcome"
3. restart app / reboot robot
4. follow unboxing flow and select wifi
check network details by tapping view network details button


# Changelog
- add fetch function


# Review requests
Please see Test plan


# Risk assessment
low
